src/bundle/Resources/views/themes/admin/ui/component/alert/alert.html.twig

```diff
@@ -28,9 +28,7 @@
         <div class="ibexa-alert__content {% if show_subtitle_below|default(false) %}ibexa-alert__content--subtitle-below{% endif %}">
             {% block title %}
                 {% if title is defined %}
-                    <div class="ibexa-alert__title">
-                        {{ title }}
-                    </div>
+                    <div class="ibexa-alert__title">{{ title }}</div>
```

Comment:
I think that adding `-` would be less error-prone approach, and prevent breaking this by mistake if someone in the future will add a new line character not knowing it cannot be here.
```suggestion
                    <div class="ibexa-alert__title">{{- title -}}</div>
```